### PR TITLE
(BNAPP-1714)handle username if so long

### PR DESCRIPTION
**Pull request by KrystianKasp98**
# Description

Fixes [BNAPP-1714](https://opuscapita.atlassian.net/browse/BNAPP-1714).

<ins>Please include a summary of the change (current and new behavior):</ins>

Shortcut for too long username.

<br/>

### 🛠️ Type of change

Please check the type of change your PR introduces:
- [ ] Feature
- [x] Bug fix
- [ ] Documentation update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

<br/>

### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

<br/>

# How to test the change locally
Just log in as user with long username E.g. `manager.commodity_de@acmemax.com`, then open user window and check results

*✍️ replace this line ✍️*

<br/>

# Other informations

### CODE REVIEW

**Review by ElaCzapiewska — CHANGES_REQUESTED** [review]
Please rethink the solution as I am not convinced. 
1. if you display anything like xyz... you should also give the possibility to view the whole string (on hoover or whatever)
2. this can also be last name that can be too long
3. this 25 chars limit and the approach used would completely destroy mine window which is very nice currently having very long data (probably the "-" helps here, but anyway...)
Screenshot from 2022-06-23 13-15-26

**Review by KacperKlosowskiOC — CHANGES_REQUESTED** [review]
Nice :) Tiny remarks only.

File: components/MainMenu/MainMenu.react.js [hunk 1]
```diff
@@ -391,6 +392,11 @@ class MainMenu extends ConditionalRenderComponent
         return tenantsForSupplierSwitch.includes(value);
     }
 
+    copyTextToClipBoard(userName) {
+        navigator.clipboard.writeText(userName);
+        this.context.showNotification(this.context.i18n.getMessage('Main.notification.copyToClipBoard',{userName}), 'info', 3);
```
Comment (ElaCzapiewska) [inline components/MainMenu/MainMenu.react.js:397]: please put a space before each next argument of a function call

File: components/MainMenu/Menu/components/MenuAccount/MenuAccount.react.js [hunk 2]
```diff
@@ -76,7 +80,7 @@ class MenuAccount extends Component
                     </div>
                     <div className="name-container">
                         <div className="full-name">{firstName} {lastName}</div>
-                        <div className="user-name">{userName}</div>
+                        <div className="user-name" title={userName} onClick={()=>copyTextToClipBoard(userName)}>{this.prepareUserName(userName)}</div>
```
Comment (ElaCzapiewska) [inline components/MainMenu/Menu/components/MenuAccount/MenuAccount.react.js:83]: please put spaces in arrow function definition

**Review by ElaCzapiewska — CHANGES_REQUESTED** [review]
Nice, only tiny formatting remarks

**Review by KacperKlosowskiOC — APPROVED** [review]
:rocket:

**Pull request merged**